src/Symfony/Component/Form/Tests/Extension/Core/Type/FileTypeTest.php

```diff
@@ -296,7 +296,7 @@ public function uploadFileErrorCodes()
     {
         return [
             'no error' => [UPLOAD_ERR_OK, null],
-            'upload_max_filesize ini directive' => [UPLOAD_ERR_INI_SIZE, 'The file is too large. Allowed maximum size is {{ limit }} {{ suffix }}.'],
+            'upload_max_filesize ini directive' => [UPLOAD_ERR_INI_SIZE, 'The file is too large. Allowed maximum size is 2 MiB.'],
```

Comment:
I think we need to add an `ini_set()` call for `upload_max_filesize` to have a deterministic failure.

Reply:
We should be able to inject a mocked translator in FileType actually, but it reveals another issue: we can't set the translator since `FormFactoryBuilder` instantiates the `CoreExtension()` with no translator, there is no way to pass one. Just had a brainstorm with @stof and @HeahDude, we might need another PR...